authors/apps/authentication/urls.py

```diff
@@ -24,4 +25,7 @@
          name='password-reset'),
     path('articles/', GetAllArticlesForCurrentUser.as_view(),
          name='personal-articles'),
+    path('article-reports/', AllUserArticleReports.as_view(),
+         name='user-article-reports')
+
```

Comment:
@BrianSerem, `article-reports/` should be inside the article's app. Not here, this is not maintainable.